deploy/sample/zfspvcr.yaml

```diff
@@ -13,5 +12,6 @@ spec:
   keylocation: ""
   ownerNodeID: zfspv-node1
   poolName: zfspv-pool
+  recordsize: 8k
   thinProvision: "off"
-
+  volumeType: DATASET
```

Comment:
can we use the terms used by the ZFS itself.. is it zvol and zfs?

Reply:
zfs dataset and zfs volume(zvol) are the terms used by zfs.

https://www.freebsd.org/doc/handbook/zfs-term.html
http://fibrevillage.com/storage/168-zfs-pool-zfs-datasets-and-zfs-volumes

Reply:
Also can this be auto determined from within the code based on the fs type. Also to keep it consistent with SC - how about adding fstype to the ZFSVolume CR instead of volumeType.

Reply:
the idea here is to store all the info related to ZFSVolume. By looking at resource we can get all the info like what is size, what are all properties that are provided while creating it, whether it is a dataset or zvol.